src/Sentry/Laravel/Tracing/Integrations/LighthouseIntegration.php

```diff
@@ -0,0 +1,174 @@
+<?php
+
+namespace Sentry\Laravel\Tracing\Integrations;
+
+use GraphQL\Language\AST\OperationDefinitionNode;
+use Illuminate\Contracts\Events\Dispatcher;
+use Nuwave\Lighthouse\Events\EndExecution;
+use Nuwave\Lighthouse\Events\EndRequest;
+use Nuwave\Lighthouse\Events\StartExecution;
+use Nuwave\Lighthouse\Events\StartRequest;
+use Sentry\Laravel\Integration;
+use Sentry\SentrySdk;
+use Sentry\Tracing\SpanContext;
+
+class LighthouseIntegration
+{
+    /** @var array<int, <string|null, \GraphQL\Language\AST\OperationDefinitionNode>> $operations */
+    private $operations;
+
+    /** @var \Sentry\Tracing\Span|null $previousSpan */
+    private $previousSpan;
+
+    /** @var \Sentry\Tracing\Span|null $requestSpan */
+    private $requestSpan;
+
+    /** @var \Sentry\Tracing\Span|null $operationSpan */
+    private $operationSpan;
+
+    public function __construct(Dispatcher $evenDispatcher)
```

Comment:
```suggestion
    public function __construct(EventDispatcher $eventDispatcher)
```

I tend to prefer aliases for those super generic Laravel class names.